### PR TITLE
[Enhancement] make cpu time more accurate

### PR DESCRIPTION
**Pull request by dirtysalt**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

### CODE REVIEW

File: be/src/connector/hive_connector.cpp [hunk 1]
```diff
@@ -71,6 +71,7 @@ Status HiveDataSource::open(RuntimeState* state) {
         _no_data = true;
         return Status::OK();
     }
+    SCOPED_TIMER(_profile.scan_timer);
```
Comment (GavinMar) [inline be/src/connector/hive_connector.cpp:74]: I'm not sure it's necessary to include scanner initial time to scan time

File: be/src/exec/vectorized/hdfs_scanner.h [hunk 2]
```diff
@@ -50,8 +51,7 @@ struct HdfsScanStats {
 
     int64_t get_cpu_time_ns() const {
         // TODO: make it more accurate
-        return expr_filter_ns + column_convert_ns + level_decode_ns + value_decode_ns + group_dict_filter_ns +
-               group_dict_decode_ns;
+        return expr_filter_ns + column_convert_ns + column_read_ns + reader_init_ns;
```
Comment (GavinMar) [inline be/src/exec/vectorized/hdfs_scanner.h:54]: The new cpu time are not overlapped?

Comment (dirtysalt) [reply be/src/exec/vectorized/hdfs_scanner.h:54]: No, not overlapped. Hoever `column_convert/decode` are included in `column_read_ns`, so `column_read_ns` is more accurated.

File: be/src/connector/hive_connector.cpp (hunk 1 above)
Comment (dirtysalt) [reply be/src/connector/hive_connector.cpp:74]: There is no harm to time it.

**Review by GavinMar — APPROVED** [review]

**Review by stephen-shelby — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by dirtysalt** [comment]
run starrocks_admit_test

**Review by Smith-Cruise — APPROVED** [review]

**Review by Youngwb — APPROVED** [review]

**Pull request merged**